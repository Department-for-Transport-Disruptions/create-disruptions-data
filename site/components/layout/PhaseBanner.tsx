import Link from "next/link";
import { ReactElement } from "react";

const PhaseBanner = (): ReactElement => (
    <div className="govuk-phase-banner max-w-[960px] w-[90%] border-0 mx-auto app-phase-banner app-width-container">
        <p className="govuk-phase-banner__content">
            <strong className="govuk-tag govuk-phase-banner__content__tag">Beta</strong>
            <span className="govuk-phase-banner__text">
                This is a new service. Help us improve it and{" "}
                <Link className="govuk-link" id="feedback-link" href="/feedback">
                    give your feedback (opens in new tab)
                </Link>
                .
            </span>
        </p>
    </div>
);

export default PhaseBanner;
