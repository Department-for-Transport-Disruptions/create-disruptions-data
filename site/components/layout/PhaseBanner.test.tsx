import { render } from "@testing-library/react";
import { describe, expect, it } from "vitest";
import PhaseBanner from "./PhaseBanner";

describe("PhaseBanner", () => {
    it("should render correctly", () => {
        const { asFragment } = render(<PhaseBanner />);
        expect(asFragment()).toMatchSnapshot();
    });
});
