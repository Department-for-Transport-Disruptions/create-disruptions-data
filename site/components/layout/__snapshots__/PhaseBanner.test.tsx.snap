// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`PhaseBanner > should render correctly 1`] = `
<DocumentFragment>
  <div
    class="govuk-phase-banner max-w-[960px] w-[90%] border-0 mx-auto app-phase-banner app-width-container"
  >
    <p
      class="govuk-phase-banner__content"
    >
      <strong
        class="govuk-tag govuk-phase-banner__content__tag"
      >
        Beta
      </strong>
      <span
        class="govuk-phase-banner__text"
      >
        This is a new service. Help us improve it and 
        <a
          class="govuk-link"
          href="/feedback"
          id="feedback-link"
        >
          give your feedback (opens in new tab)
        </a>
        .
      </span>
    </p>
  </div>
</DocumentFragment>
`;
